src/seyifunmiSF/currency.go

```diff
@@ -0,0 +1,53 @@
+package seyifunmiSF
+
+type NGN float32 //Nigeran naira
+type JPY float32 //Japanese Yen
+type USD float32 //United States Dollar
+type AUD float32 //Australia Dollar
+type GBP float32 //Great British Pound
+type ARS float32 //Argentine peso
+type BRL float32 //Brazillian Real
+type CAD float32 //Canadian Dollar
+type ZAR float32 //South African Rand
+type EUR float32 //Euro
+type AED float32 //United Arab Emirates
+
+func (n NGN) JPY() JPY {
+	return JPY(n * 0.34134)
+}
+
```

Comment:
@seyifunmiSF, nice. You may add comments to exported methods and tag your instructor to review your code.